viscy/data/hcs.py

```diff
@@ -318,6 +324,7 @@ def __init__(
         augmentations: list[MapTransform] = [],
         caching: bool = False,
         ground_truth_masks: Path | None = None,
+        pyramid_resolution: str = "0",
```

Comment:
#218 used `array_key` for this parameter, which is a more general naming:

https://github.com/mehta-lab/VisCy/blob/1d3261d1e592e1a0c518670295dc6c84ee0806fb/viscy/data/mmap_cache.py#L40